quantecon/_inequality.py

```diff
@@ -108,7 +108,8 @@ def shorrocks_index(A):
     ----------
     .. [1] Wealth distribution and social mobility in the US:
        A quantitative approach (Benhabib, Bisin, Luo, 2017).
-       https://www.econ.nyu.edu/user/bisina/RevisionAugust.pdf
+       https://www.aeaweb.org/articles?id=10.1257/aer.20151684
```

Comment:
Update "Benhabib, Bisin, Luo, 2017" as well? (to publication information in AER)